Add FIXME in `CSS*` IDL files and comments

Ahmad-S792 opened the pull request:
#### 24afea6105eb95fb3cf0acb2509cc7eec1242157
<pre>
Add FIXME in `CSS*` IDL files and comments

<a href="https://bugs.webkit.org/show_bug.cgi?id=265327">https://bugs.webkit.org/show_bug.cgi?id=265327</a>

Reviewed by Anne van Kesteren.

This patch is to add FIXME links and web specification URLs
for quick reference.

* Source/WebCore/css/CSSConditionRule.idl:
* Source/WebCore/css/CSSContainerRule.idl:
* Source/WebCore/css/CSSFontFaceRule.idl:
* Source/WebCore/css/CSSFontFeatureValuesRule.idl:
* Source/WebCore/css/CSSFontPaletteValuesRule.idl:
* Source/WebCore/css/CSSGroupingRule.idl:
* Source/WebCore/css/CSSImportRule.idl:
* Source/WebCore/css/CSSKeyframeRule.idl:
* Source/WebCore/css/CSSKeyframeRule.idl:
* Source/WebCore/css/CSSLayerBlockRule.idl:
* Source/WebCore/css/CSSLayerStatementRule.idl:
* Source/WebCore/css/CSSMediaRule.idl:
* Source/WebCore/css/CSSPageRule.idl:
* Source/WebCore/css/CSSPropertyRule.idl:
* Source/WebCore/css/CSSRule.idl:
* Source/WebCore/css/CSSRuleList.idl:
* Source/WebCore/css/CSSScopeRule.idl:
* Source/WebCore/css/CSSStyleDeclaration.idl:
* Source/WebCore/css/CSSStyleSheet.idl:
* Source/WebCore/css/CSSSupportsRule.idl:
* Source/WebCore/css/DOMCSSCustomPropertyDescriptor.idl:
* Source/WebCore/css/DOMMatrix.idl:
* Source/WebCore/css/DOMMatrix2DInit.idl:
* Source/WebCore/css/DOMMatrixInit.idl:
* Source/WebCore/css/FontFace.idl:
* Source/WebCore/css/FontFaceSet.idl:
* Source/WebCore/css/LinkStyle.idl:
* Source/WebCore/css/MediaList.idl:
* Source/WebCore/css/MediaQueryList.idl:
* Source/WebCore/css/MediaQueryListEvent.idl:
* Source/WebCore/css/StyleSheet.idl:
* Source/WebCore/css/StyleSheetList.idl:

Canonical link: <a href="https://commits.webkit.org/277518@main">https://commits.webkit.org/277518@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/72312efe9b115af2e57a1d247b69725519a0bd23

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/47878 "Passed style check") | [  ~~🛠 ios~~](https://ews-build.webkit.org/#/builders/48/builds/27075 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac~~](https://ews-build.webkit.org/#/builders/14/builds/50715 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/5/builds/50559 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/43931 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [❌ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/50184 "Failed to checkout and rebase branch from PR 27155") | [  ~~🛠 ios-sim~~](https://ews-build.webkit.org/#/builders/49/builds/32946 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/24561 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/5/builds/50559 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/48460 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/49/builds/32946 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/14/builds/50715 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/5/builds/50559 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/49/builds/32946 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/14/builds/50715 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe-skia~~](https://ews-build.webkit.org/#/builders/52/builds/5925 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/49/builds/32946 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/14/builds/50715 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/2/builds/52453 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/44/builds/22920 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/50/builds/19268 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/2/builds/52453 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/46/builds/24190 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/14/builds/50715 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/2/builds/52453 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/43/builds/24979 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/6773 "Built successfully and passed tests") | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/45/builds/23909 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

annevk commented on Source/WebCore/css/FontFaceSet.idl:
```diff
@@ -37,6 +39,7 @@ enum FontFaceSetLoadStatus {
     boolean has(FontFace font);
 
     // FIXME: We should add support for the setlike declaration.
+    // https://bugs.webkit.org/show_bug.cgi?id=230119
```
Nit:
```suggestion
    // FIXME: We should add support for the setlike declaration (webkit.org/b/230119).
```

Ahmad-S792 replied:
Done. :-)

Ahmad-S792 committed:
Add FIXME in `CSS*` IDL files and comments

https://bugs.webkit.org/show_bug.cgi?id=265327

Reviewed by Anne van Kesteren.

This patch is to add FIXME links and web specification URLs
for quick reference.

* Source/WebCore/css/CSSConditionRule.idl:
* Source/WebCore/css/CSSContainerRule.idl:
* Source/WebCore/css/CSSFontFaceRule.idl:
* Source/WebCore/css/CSSFontFeatureValuesRule.idl:
* Source/WebCore/css/CSSFontPaletteValuesRule.idl:
* Source/WebCore/css/CSSGroupingRule.idl:
* Source/WebCore/css/CSSImportRule.idl:
* Source/WebCore/css/CSSKeyframeRule.idl:
* Source/WebCore/css/CSSKeyframeRule.idl:
* Source/WebCore/css/CSSLayerBlockRule.idl:
* Source/WebCore/css/CSSLayerStatementRule.idl:
* Source/WebCore/css/CSSMediaRule.idl:
* Source/WebCore/css/CSSPageRule.idl:
* Source/WebCore/css/CSSPropertyRule.idl:
* Source/WebCore/css/CSSRule.idl:
* Source/WebCore/css/CSSRuleList.idl:
* Source/WebCore/css/CSSScopeRule.idl:
* Source/WebCore/css/CSSStyleDeclaration.idl:
* Source/WebCore/css/CSSStyleSheet.idl:
* Source/WebCore/css/CSSSupportsRule.idl:
* Source/WebCore/css/DOMCSSCustomPropertyDescriptor.idl:
* Source/WebCore/css/DOMMatrix.idl:
* Source/WebCore/css/DOMMatrix2DInit.idl:
* Source/WebCore/css/DOMMatrixInit.idl:
* Source/WebCore/css/FontFace.idl:
* Source/WebCore/css/FontFaceSet.idl:
* Source/WebCore/css/LinkStyle.idl:
* Source/WebCore/css/MediaList.idl:
* Source/WebCore/css/MediaQueryList.idl:
* Source/WebCore/css/MediaQueryListEvent.idl:
* Source/WebCore/css/StyleSheet.idl:
* Source/WebCore/css/StyleSheetList.idl:

Canonical link: https://commits.webkit.org/277518@main
